### PR TITLE
 Fix EEFNameSpacedPropertyEditorDialog issues in Mediators

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
 Fixing EEFNameSpacedPropertyEditorDialog issues in:
- SendMediator
- XSLTMediator
- SequenceMediator
- SmooksMediator
- StoreMediator
- SwitchMediator
- ValidateMediator

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SendMediatorPropertiesEditionPartForm.java [hunk 1]
```diff
@@ -845,32 +847,28 @@ protected Composite createDynamicReceivingSequence(FormToolkit widgetFactory, fi
...
-						.firePropertiesChanged(new PropertiesEditionEvent(SendMediatorPropertiesEditionPartForm.this,
-								EsbViewsRepository.SendMediator.Properties.dynamicReceivingSequence,
-								PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null,
-								getDynamicReceivingSequence()));
-			}
-		});
+		dynamicReceivingSequenceText.addMouseListener(new MouseAdapter() {
+            
+            @Override
+            public void mouseDown( MouseEvent event ) {
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SendMediatorPropertiesEditionPartForm.java]: Fix formatting issues.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java [hunk 2]
```diff
@@ -1287,32 +1290,28 @@ protected Composite createDynamicReferenceKey(FormToolkit widgetFactory, final C
...
-				// valueExpression.setPropertyValue(valueExpressionText.getText());
-				nspd.open();
-				dynamicReferenceKeyText.setText(dynamicReferenceKey.getPropertyValue());
-				propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(
-						SequencePropertiesEditionPartForm.this,
-						EsbViewsRepository.Sequence.Properties.dynamicReferenceKey,
-						PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getdDynamicReferenceKey()));
-			}
-		});
+		dynamicReferenceKeyText.addMouseListener(new MouseAdapter() {
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java]: Fix formatting issues.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java [hunk 3]
```diff
@@ -1326,6 +1325,18 @@ public void focusGained(FocusEvent e) {
 		return parent;
 	}
 	
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, dynamicReferenceKey);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java]: Remove commented line.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SwitchMediatorPropertiesEditionPartForm.java [hunk 4]
```diff
@@ -971,35 +974,46 @@ protected Composite createSourceXPathWidget(FormToolkit widgetFactory, final Com
...
        EditingUtils.setEEFtype(sourceXPathText, "eef::Text");
        Control itemHelp = FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent.getHelpContent(EsbViewsRepository.SwitchMediator.Properties.sourceXPath, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
        sourceXPathElements = new Control[] {itemLabel, sourceXPathText, itemHelp};
        return parent;
     }
    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(), SWT.NULL,
+                sourceXPath);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SwitchMediatorPropertiesEditionPartForm.java]: Remove commented line.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ValidateMediatorPropertiesEditionPartForm.java [hunk 5]
```diff
@@ -960,39 +963,48 @@ protected Composite createSourceWidget(FormToolkit widgetFactory, final Composit
...
         Control sourceXPathHelp = FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent
                 .getHelpContent(EsbViewsRepository.ValidateMediator.Properties.source, EsbViewsRepository.FORM_KIND),
                 null); // $NON-NLS-1$
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, source);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ValidateMediatorPropertiesEditionPartForm.java]: Remove commented line.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SwitchMediatorPropertiesEditionPartForm.java (hunk 4 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SwitchMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/1d53e06b88e566a6d316666c06dd48202d396d30

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ValidateMediatorPropertiesEditionPartForm.java (hunk 5 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ValidateMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/1d53e06b88e566a6d316666c06dd48202d396d30

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java (hunk 3 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java (hunk 2 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SendMediatorPropertiesEditionPartForm.java (hunk 1 above)
Comment (prabushi) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SendMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

**Pull request merged**